data/card_definitions.json

```diff
@@ -19359,6 +19359,317 @@
 	},
 
 
+	{
+		"NOTE": "ZANGIEF CARDS START",
+		"id": "zangief_ultimateatomicbuster",
+		"type": "ultra",
+		"display_name": "Ultimate Atomic Buster",
+		"force_cost": 0,
+		"gauge_cost": 4,
+		"range_min": 1,
+		"range_max": 1,
+		"speed": 5,
+		"power": 11,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "hit",
+				"effect_type": "discard_hand",
+				"and": {
+					"effect_type": "opponent_discard_hand"
+				}
+			}
+		],
+		"boost": {
+			"boost_type": "continuous",
+			"force_cost": 0,
+			"cancel_cost": -1,
+			"display_name": "Super Armor",
+			"effects": [
+				{
+					"timing": "during_strike",
+					"effect_type": "powerup",
+					"amount": 1,
+					"and": {
+						"effect_type": "stun_immunity"
+					}
+				},
+				{
+					"timing": "after",
+					"condition": "was_hit",
+					"effect_type": "add_boost_to_gauge_on_strike_cleanup"
+				}
+			]
+		}
+	},
+	{
+		"id": "zangief_siberianblizzard",
+		"type": "ultra",
+		"display_name": "Siberian Blizzard",
+		"force_cost": 0,
+		"gauge_cost": 3,
+		"range_min": 1,
+		"range_max": 1,
+		"speed": 2,
+		"power": 8,
+		"armor": 0,
+		"guard": 6,
+		"effects": [
+			{
+				"timing": "during_strike",
+				"effect_type": "opponent_cant_move_past"
+			},
+			{
+				"timing": "now",
```

Comment:
this probably doesn't need the now timing effect since it's on an attack rather than a boost (does the discarded timing work for attacks? should it be cleanup instead?)

Reply:
hmm, I think putting remove on cleanup is probably what we want also?